src/OrchardCore.Modules/OrchardCore.Liquid/Filters/LiquidFilter.cs

```diff
@@ -25,7 +28,7 @@ public async ValueTask<FluidValue> ProcessAsync(FluidValue input, FilterArgument
                 context.SetValue("Model", model);
             }
 
-            var content = await _liquidTemplateManager.RenderAsync(input.ToStringValue(), context);
+            var content = await _liquidTemplateManager.RenderAsync(input.ToStringValue(), _htmlEncoder, context);
 
```

Comment:
Basically agree with what @jtkech has said, and also confused about what should be encoded vs what shouldn't.

Think it makes sense to move all the liquid rendering to use the injected `HtmlEncoder`, and get rid of the extension method override that was being used here. 

The `Html.Raw()` in the Html template is a bit questionable though, not sure.
I would ask the same question about the agency templates where I used 
`{{ Model.ContentItem.Content.RawHtml.Content.Html | liquid | raw }}` and was not entirely sure that it needed the `raw` pipe (and I think you questioned that in the meeting when going through PR's)

Reply:
If the property contains Html then you should use ` | raw`.

Reply:
Otherwise you'll get all tags encoded and be rendered as is, kind of obvious while I am stating that.